lib/solveEquation/stepThrough.js

```diff
@@ -1,17 +1,15 @@
 'use strict';
 
+const ChangeTypes = require('../ChangeTypes');
+const checks = require('../checks');
 const Equation = require('../equation/Equation');
-const EquationStatus = require('../equation/Status');
 const EquationOperations = require('./EquationOperations');
```

Comment:
I kinda liked how the equation requires were grouped together, though I'm not that set on it